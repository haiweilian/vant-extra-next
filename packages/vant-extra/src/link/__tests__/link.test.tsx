import { nextTick } from 'vue'
import { mount } from '@vue/test-utils'
import { describe, test, expect } from 'vitest'
import { Link } from '../index'

describe('Link', () => {
  test('render', async () => {
    const wrapper = mount(Link)
    await nextTick()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
