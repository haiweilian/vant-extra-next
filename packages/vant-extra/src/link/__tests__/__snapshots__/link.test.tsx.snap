// Vitest Snapshot v1

exports[`Link > render 1`] = `"<a class=\\"vae-link vae-link--default vae-link--underline-always\\"></a>"`;
